apcd-cms/src/apps/components/paginator/static/paginator/css/paginator.css

```diff
@@ -0,0 +1,70 @@
+/*! @tacc/core-styles 1.0.0+ | MIT | github.com/TACC/Core-Styles */
+.c-page-list,.pagination {
+    --item-width:2em;
+    --item-buffer:3px;
+    --item-border:var(--global-border-width--normal)
+}
+.c-page-list,.pagination {
+    align-items:baseline;
+    justify-content: center;
+    padding-top: 20px;
+    display:flex;
+    flex-direction:row;
+    font-size:14px;
+    list-style:none
+}
+ul.c-page-list {
+    list-style:none
+}
+.c-page-link--always-click {
+    box-sizing:content-box;
+    position:relative
+}
+.c-page-link--always-click:after,.c-page-link--always-click:before {
+    --width:var(--item-buffer);
+    --vert-offset:calc(var(--item-border)*-1);
+    --horz-offset:calc(100% + var(--item-border));
+    bottom:var(--vert-offset);content:"";position:absolute;
+    top:var(--vert-offset);width:var(--width)
+}
+.c-page-link--always-click.c-button {
+    overflow:visible
+}
+.c-page-link--always-click:before {
+    right:var(--horz-offset)
+}
+.c-page-link--always-click:after {
+    left:var(--horz-offset)
+}
+.c-page-item {
+    margin-inline:var(--item-buffer)
+}
+.c-page-item:first-child {
+    margin-left:0
+}
+.c-page-item:last-child {
+    margin-right:0
+}
+.c-page-item--etcetera {
+    cursor:default;
+    margin-inline:2px;
+    min-width:var(--item-width);
+    text-align:center
+}
+.c-page-end,.pagination a,.pagination span {
+    padding:4px 12px
+}
+.c-page-link {
+    line-height:1.2;
+    min-width:var(--item-width);
+    padding:5px 0
+}
+.pagination {
+    margin-block:var(--global-space--x-large);
+    margin-inline:auto;
+    width:-moz-fit-content;
+    width:fit-content
+}
+/*.pagination a,.pagination span {
+    padding-block:0
+}*/
```

Comment:
**Undesired Opportunity for Untracked Divergence from Source.** By pasting in an _un-minified_ stylesheet, you've suggested to other developers that these styles are APCD-specific. Which then leads to edits to the styles that are not tracked as differences with the source.

Please treat these styles like external third-party code[^1], and customize them using (A) the available customization options and/or (B) overwriting the styles.

<details><summary>A. <strong>Available Customization</strong></summary>

I.e. [CSS Custom Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*). Not suitable for the removal of `padding-block`, so see **B**.
```css
    --item-width
    --item-buffer
    --item-border
```

</details>
<details><summary>B. <strong>Overwriting Styles</strong></summary>

There are multiple ways to do this clearly. Evaluate the URL to use in these examples, after reading [my other comment about these styles].

1. ```css
    /*! @tacc/core-styles 0.10.0+ | MIT | github.com/TACC/Core-Styles */.c-page-list,.pagination{--item-width:2em;--item-buffer:3px;--item-border:var(--global-border-width--normal)}.c-page-list,.pagination{align-items:baseline;display:flex;flex-direction:row;font-size:14px;list-style:none}ul.c-page-list{list-style:none}.c-page-link--always-click{box-sizing:content-box;position:relative}.c-page-link--always-click:after,.c-page-link--always-click:before{--width:var(--item-buffer);--vert-offset:calc(var(--item-border)*-1);--horz-offset:calc(100% + var(--item-border));bottom:var(--vert-offset);content:"";position:absolute;top:var(--vert-offset);width:var(--width)}.c-page-link--always-click.c-button{overflow:visible}.c-page-link--always-click:before{right:var(--horz-offset)}.c-page-link--always-click:after{left:var(--horz-offset)}.c-page-item{margin-inline:var(--item-buffer)}.c-page-item:first-child{margin-left:0}.c-page-item:last-child{margin-right:0}.c-page-item--etcetera{cursor:default;margin-inline:2px;min-width:var(--item-width);text-align:center}.c-page-end,.pagination a,.pagination span{padding:4px 12px}.c-page-link{line-height:1.2;min-width:var(--item-width);padding:5px 0}.pagination{margin-block:var(--global-space--x-large);margin-inline:auto;width:-moz-fit-content;width:fit-content}.pagination a,.pagination span{padding-block:0}

    .pagination a,
    .pagination span {
        padding-block: unset; /* or some other suitable value */
    }
    ```
2. ```css
    @import url("https://unpkg.com/browse/@tacc/core-styles@0.10.0/dist/components/bootstrap.pagination.css");

    .pagination a,
    .pagination span {
        padding-block: unset; /* or some suitable value */
    }
    ```

To learn more about `unset` and friends, see https://css-tricks.com/inherit-initial-unset-revert/.

</details>

[^1]: These are from an outside library: https://www.npmjs.com/package/@tacc/core-styles.

Reply:
It was my original idea to import an un-minified stylesheet, so I caused this "undesired" thing 😅.

At the time, I was unaware that unpkg.com had all npm package version files available.